### PR TITLE
Adding some helper scripts

### PR DESCRIPTION
**Pull request by rhuttman**

### CODE REVIEW

File: scripts/run-pallet-benchmarks.sh [hunk 1]
```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+read -p 'Pallet name: ' pallet
+
+pallet_name=$(echo "$pallet" | tr '-' '_')
```
Comment (atomaka) [inline scripts/run-pallet-benchmarks.sh:5]:
```suggestion
PS3="Pallet name: "
select pallet_name in $(ls pallets | tr '-' '_'); do break; done 
 ```

If you're feeling fancy.

File: scripts/run-pallet-benchmarks.sh [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+cargo run \
+--release \
```
Comment (atomaka) [inline scripts/run-pallet-benchmarks.sh:8]: Generally see commands continued on the next line indented. ie.

```
cargo run \
  --release
```

**Review by atomaka — APPROVED** [review]
Minor comments; nothing blocking.

File: scripts/run-pallet-benchmarks.sh (hunk 2 above)
Comment (rhuttman) [reply scripts/run-pallet-benchmarks.sh:7]: I was wondering about this

**Pull request merged**